### DISCUSSION TITLE
Migration from api-tools to DK API

### DISCUSSION THREAD

**Discussion by arhimede** [dotkernel/api#301] [General]
documentation or guidance

and maybe a tool

**Comment by arhimede** [comment]
warning:
API Tools doesn't use Doctrine out of the box, though,  - out of the box it uses laminas-db...

**Comment by arhimede** [comment]
1. study api-tools and see what features have it and DK API doess not have  ( comparative list) rest, auth, etc.
2. **document** how to migrate from one to another 
3. how to implement Core architecture

**Reply by diego-sorribas** [reply]
Hey there! Is there a migration guide from Api Tools?
